src/main/java/org/jenkinsci/plugins/workflow/cps/CpsThread.java

```diff
@@ -160,9 +194,13 @@ public StepExecution getStep() {
 
         try (Timeout timeout = Timeout.limit(5, TimeUnit.MINUTES)) {
             LOGGER.log(FINE, "runNextChunk on {0}", resumeValue);
-            Outcome o = resumeValue;
+            final Outcome o = resumeValue;
             resumeValue = null;
-            outcome = program.run0(o);
+            outcome = GroovyCategorySupport.use(Arrays.<Class>asList(CpsDefaultGroovyMethods.class, CpsDefaultGroovyMethodsExt.class), new Closure<Outcome>(null) {
+                @Override public Outcome call() {
+                    return program.run0(o);
```

Comment:
This is just for experimentation; probably we would rather want to have a change in `groovy-cps` which

* fleshes out `CpsDefaultGroovyMethods`
* calls `GroovyCategorySupport.use` from inside `Continuable.run0`